sklearn/tree/_criterion.pyx

```diff
@@ -247,22 +257,18 @@ cdef class ClassificationCriterion(Criterion):
         self.label_count_stride = label_count_stride
 
         cdef SIZE_t n_elements = n_outputs * label_count_stride
-        self.label_count_left = <double*> calloc(n_elements, sizeof(double))
-        self.label_count_right = <double*> calloc(n_elements, sizeof(double))
-        self.label_count_total = <double*> calloc(n_elements, sizeof(double))
+        self.node_sum_left = <double*> calloc(n_elements, sizeof(double))
+        self.node_sum_right = <double*> calloc(n_elements, sizeof(double))
+        self.node_sum_total = <double*> calloc(n_elements, sizeof(double))
 
-        if (self.label_count_left == NULL or
-                self.label_count_right == NULL or
-                self.label_count_total == NULL):
+        if (self.node_sum_left == NULL or self.node_sum_right == NULL or
+            self.node_sum_total == NULL):
```

Comment:
can you keep one per line? it was clearer.

You have also that the line 265 doesn't distinguish from the next on.